src/renderers/shared/fiber/__tests__/ReactIncrementalSideEffects-test.js

```diff
@@ -831,9 +831,9 @@ describe('ReactIncrementalSideEffects', () => {
         // Updated.
         span(1),
         div(
+          span(1),
```

Comment:
I think this change is correct. The effect is flushed sooner than before because we're reusing more work. I may be wrong.